ios/FluentUI/Tooltip/TooltipView.swift

```diff
@@ -159,12 +161,16 @@ class TooltipView: UIView {
 
     private func updateWindowSpecificColors() {
         if let window = window {
```

Comment:
but if we don't need branding color, we don't need to check window anymore? there is no use?

Reply:
Why do we need to check window with branding colors?

Reply:
great question. our framework is setup so that branding color is associated with UIWindow. Clients can set different branding color per UIWindow. https://github.com/microsoft/fluentui-apple/blob/e0d2f34d07d7ccb04d11af9ccc36a6b0db824d2e/ios/FluentUI/Core/Colors.swift#:~:text=public%20protocol-,ColorProviding,-%7B